source/stylesheets/_variables.scss

```diff
@@ -20,7 +20,7 @@ under the License.
 ////////////////////////////////////////////////////////////////////////////////
 // Use these settings to help adjust the appearance of Slate
 
-@import 'http://cheddargetter.com/app/fonts/TradeGothic.css';
+@import 'http://getcheddar.com/app/fonts/TradeGothic.css';
```

Comment:
Actually I don't think this will be a problem. We're already (somehow) adding the `Access-Control-Allow-Origin:*` header when serving this file. If it's a problem, we can fix it later.